src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,223 @@
+$base-color: #000066;
+$white-bg: #fff;
+$link-color: #939393;
+
+.list-item {
+  padding: 2em 1em;
+  display: flex;
+  flex-wrap: wrap;
+  box-shadow: 0 1px 8px 1px rgba(0, 0, 0, 0.1);
+  border-radius: 10px;
+  position: relative;
+  background-color: $white-bg;
+  color: $base-color;
+}
+
+.profile-details {
+  padding-left: 150px;
+  padding-top: 20px;
+  flex-basis: 50%;
+  max-width: 50%;
+  position: relative;
+}
+
+.avatar-div {
+  left: 0;
+  position: absolute;
```

Comment:
Please don't use `position: absolute;  left: 0; top: 10px;` I don't understand their use-case they are generally helpful in layering content over each other like if we have to use `z-index` etc.

What is the use-case of this in this design?

Reply:
The use of position is to fix the size of image. Making it flex will shrink it according to size of width. That is why position is used

Reply:
@ashishnagpal2498 which Images are you taking about, you have used the same for `svgs` as well.